README.md

```diff
@@ -64,6 +64,7 @@ The following steps are just an example of how to run the application in a produ
 
 * Python 3.4 or higher
 * Elasticsearch 6.x
+* MySQL or SQLite
```

Comment:
My advise here is don't tell people to use MySQL until you have a build in CI verifying that it works properly. It's ok to use SQLite as long as it does not become a bottleneck.